legacy/tests/conftest.py

```diff
@@ -0,0 +1,7 @@
+import flask
+import pytest
+
+
+@pytest.fixture
+def runner():
+    return flask.Flask("test").test_cli_runner()
```

Comment:
The old version sets up a whole Conbench Flask app for this purpose, but this minimal version seems to work fine. I think `click` has testing utils we could switch to instead, but switching doesn't seem worth the time.